webapp/app.py

```diff
@@ -0,0 +1,92 @@
+# Code
+import datetime
+import glob
+import json
+import os
+
+# Packages
+import flask
+from canonicalwebteam.flask_base.app import FlaskBase
+from canonicalwebteam.templatefinder import TemplateFinder
+
+
+# Rename your project below
```

Comment:
What should I rename? :P

Reply:
Removed in https://github.com/canonical-web-and-design/vanilla-framework/pull/2807